docs/ref/contrib/sitemaps.txt

```diff
@@ -530,3 +545,19 @@ Once the sitemaps application is added to your project, you may also
 ping Google using the ``ping_google`` management command::
 
     python manage.py ping_google [/sitemap.xml]
+
+The management command accepts the following optional arguments:
+
+.. django-admin-option:: --site-domain SITE_DOMAIN
+
+.. versionadded:: 2.2
+
+Specifies the site domain to be used if ``django.contrib.sites`` is not used.
```

Comment:
I don't understand this. You need to put django.contrib.sites in INSTALLED_APPS in order to use the ping_google command.